R/article.R

```diff
@@ -444,3 +444,13 @@ trb_article <- function(..., keep_tex = TRUE, citation_package = 'natbib') {
     "trb", keep_tex = keep_tex, citation_package = citation_package, ...
   )
 }
+
+#' @section \code{wellcomeor_article}: Format for creating submissions to
```

Comment:
Just that we can use markdown in roxygen comment now https://roxygen2.r-lib.org/articles/rd-formatting.html#links-2

```suggestion
#' @section `wellcomeor_article`: Format for creating submissions to
```